.github/workflows/ci.yml

```
@@ -6,186 +6,130 @@ on:
           generateSarif: "1"
-      - name: Upload Semgrep SARIF results
+          sarifFile: semgrep.sarif
+      - name: Verify SARIF file exists
+        run: |
+          if [ -f "semgrep.sarif" ]; then
+            echo "✅ SARIF file found: semgrep.sarif"
+            ls -la semgrep.sarif
+          else
+            echo "⚠️  SARIF file not found, creating empty one"
+            cat > semgrep.sarif <<EOF
+{
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 102: could not find expected ':'

(syntax-check)